controllers/util/storageconsumer.go

```diff
@@ -384,6 +392,31 @@ func GetStorageRequestName(consumerUUID, storageClaimName string) string {
 	return fmt.Sprintf("storagerequest-%s", getStorageRequestHash(consumerUUID, storageClaimName))
 }
 
+func GetDestToSrcClassMapping[T CommonClassEntries](entries []T) map[string]string {
```

Comment:
This is a meanigless name, the words `Dest` and `src` do not mean anything. 
Please use a name the is meaningful, something like: `GetClassNameMapping`